Doc/library/enum.rst

```diff
@@ -547,7 +561,7 @@ Data Types
          ...     CRIMSON = 1
          Traceback (most recent call last):
          ...
-         ValueError: aliases found in <enum 'Color'>: CRIMSON -> RED
+         ValueError: aliases found in <enum 'Color'>:  CRIMSON -> RED
```

Comment:
Is it GitHub, or are there two spaces? Is it intentional?

Reply:
There are two, should only be one.

Reply:
Fixed.